tools/src/commands/GenerateDocsAPIData.ts

```diff
@@ -49,7 +50,13 @@ const executeCommand = async (
   if (project) {
     await app.generateJson(project, jsonOutputPath);
     if (MINIFY_JSON) {
-      await fs.writeFile(jsonOutputPath, JSON.stringify(await fs.readJson(jsonOutputPath), null, 0));
+      await fs.writeFile(
+        jsonOutputPath,
+        JSON.stringify(recursiveOmitBy(
+          await fs.readJson(jsonOutputPath),
+          ({ key }) => key === 'id' || key === 'groups'
+        ), null, 0)
+      );
```

Comment:
I think this would be a little bit more clear — too many indents and nested method calls makes the code harder to read 🙂 
```suggestion
      const minifiedJson = recursiveOmitBy(
        await fs.readJson(jsonOutputPath),
        ({ key }) => key === 'id' || key === 'groups'
      );
      await fs.writeFile(jsonOutputPath, JSON.stringify(minifiedJson, null, 0));
```

Reply:
To be 100% accurate, `JSON.stringify` is also a part of minifying process, but I agree that the suggestion readability looks better! 🙂 Probably the code will be changed and tweaked in the future, so there will be more opportunities to reformat it.